Android/app/src/go/doh/doh.go

```diff
@@ -477,7 +494,7 @@ func (t *transport) GetURL() string {
 
 // Perform a query using the transport, and send the response to the writer.
 func forwardQuery(t Transport, q []byte, c io.Writer) error {
```

Comment:
Don't we need to wire the context to this function as well?

Reply:
As discussed offline, we need to fix the lwipDevice in Outline SDK first: https://github.com/Jigsaw-Code/outline-sdk/blob/main/network/lwip2transport/tcp.go#L39